V6_MIGRATION.md

```diff
@@ -32,8 +32,6 @@ carthage update --use-xcframeworks
 ## Venmo
 `BTVenmoDriver` has been renamed to `BTVenmoClient`
 
-`BTVenmoRequest` must now be initialized with a `paymentMethodUsage`. 
```

Comment:
Do we want to keep this for the v6 migration guide?

Reply:
ahh yes great catch! addressed in 0d22379d61a820387ce1ed2143516a7bb87072ce